cmd/tether/tether.go

```diff
@@ -215,10 +216,17 @@ func handleSessionExit(session *SessionConfig) error {
 func launch(session *SessionConfig) error {
 	defer trace.End(trace.Begin("launching session " + session.ID))
 
+	// encode the result whether success or error
+	defer func() {
```

Comment:
If you do that then the session.Started value on entry to Launch is used and not it's value on exit. Made this mistaken initially ;)